components/atom/tooltip/src/index.js

```diff
@@ -61,11 +61,18 @@ class AtomTooltip extends Component {
     return React.Children.map(childrenOnly, child => {
       this.onClickTarget = child.props.onClick
       this.title = child.props.title
-      return React.cloneElement(child, {
-        ref,
-        className,
-        onTouchEnd
-      })
+
+      return typeof child?.type !== 'string'
+        ? React.createElement(
+            'div',
+            {
+              ref,
+              className: cx(className, `${className}--wrapper`),
```

Comment:
I think that you don't need `classnames` to concatenate two classes.

```suggestion
              className: `${className} ${className}--wrapper`,
```

Reply:
yes, is not necessary used with CX, thanks 🤗